packages/cli/src/cli/index.ts

```diff
@@ -34,6 +38,9 @@ program
   )
   .version(getVersion(), "-v, --version", "Display the version number")
   .action(async (options) => {
+    if (endpointId) {
+      options.endpointId = endpointId;
+    }
     await initCommand(options);
```

Comment:
Can you move this code into the `initCommand` (specifically here: https://github.com/triggerdotdev/trigger.dev/blob/2d722b94bd448e39506f0d6c9fff3c83e373ef2b/packages/cli/src/commands/init.ts#L203), and please make use of `readJSONFile` from `utils/fileSystem`. Also make sure to use the `resolvedPath`

Reply:
Awesome! I'll do this too.

Reply:
How does this look? 

```ts
    if (!options.endpointSlug) {
      resolvedOptions.endpointSlug = await promptEndpointSlug(path);
      const resolvedPath = resolvePath(options.projectPath);

      const packageJSONPath = pathModule.join(resolvedPath, "package.json");
      const packageJSON = await readJSONFile(packageJSONPath);

      if (packageJSON && packageJSON.endpointId) {
        options.endpointSlug = packageJSON.endpointId;
      }
    }
   ```

Reply:
Close! You should look for an endpointSlug before doing the `promptEndpointSlug` call, and then skip the`promptEndpointSlug` call if the endpointId is found in the package.json

Reply:
Aha! Great! I'll tkae that into consideration now. 

Just to digress a bit. Have you had a look at #170? I added some changes a while ago.

Reply:
Just left another review on #170 👍